src/en/news/blog/2022/improving-dashboard-accessibility-for-blind-and-visually-impaired-ceph-users/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Improving dashboard accessibility for blind and visually impaired ceph users"
+date: "2022-09-09"
+author: "Ngwa Sedrick Meh"
+image: "images/banner.png"
+tags:
+  - "ceph"
+  - "dashboard"
+  - "accessibility"
+  - "a11y"
+  - "wcag2.1"
+  - "testing"
+  - "internship"
+  - "gsoc"
+---
+
+## Introduction
+
+Web accessibility, or e-accessibility, is the inclusive practice of ensuring there are no barriers
+that prevent interaction with, or access to, websites on the World Wide Web by people with
+physical disabilities, situational disabilities, and socio-economic restrictions on bandwidth and
+speed. When sites are correctly designed, developed and edited, more users have equal access
+to information and functionality. ([Wikipedia](https://en.wikipedia.org/wiki/Web_accessibility))
+
+The Ceph Dashboard is a built-in web-based Ceph management and monitoring application through which you can inspect and administer various aspects and resources within the cluster.
+
+## Problem description
+
+While it's very likely that the Ceph dashboard meets some accessibility guidelines (ARIA attributes, alt text, etc), we currently ignore its level of conformance to WCAG 2.1 standard (levels A, AA and AAA).
+
+In a recent article published by the world bank on [disability inclusion](https://www.worldbank.org/en/topic/disability#1), disabilities affect about 1
+billion people world wide. These people use web technologies in a variety of ways. Some
+primarily navigate sites by means of a keyboard alone, while others with vision disabilities rely
+on assistive technologies like screen readers.
+
+By ensuring the Dashboard meets WCAG 2.1 level A compliance, blind and visually impaired Ceph users will not be left out and will be able to manage their Ceph clusters comfortably.
+
+## Project
+
+For the past few months we have been working on updating the dashboard to meet WCAG 2.1 level A conformance so that blind and visually impaired Ceph users can also use it comfortably to manage and monitor their Ceph clusters.
+
+To achieve this goal, we had to:
+
+1. Test and identify areas of the dashboard in need of an accessibility update
+2. Apply fixes based on WCAG 2.1 guidelines
+3. Add automated testing to prevent regressions
+
+### Identifying accessibility issues
+
+After testing several recommended [accessibilty checkers](https://www.w3.org/WAI/ER/tools/?q=wcag-21-w3c-web-content-accessibility-guidelines-21) by the W3C, we chose the [accessibility insights plugin](https://accessibilityinsights.io/docs/en/web/overview/) to help quickly identify the most common accessibility issues for each page of the the dashboard.
+
+### Accessibility improvements
+
+Based on the identified issues, we improved the accessibility of several key components including:
+
+#### Login Page
+
+By improving the color contrast of the login form, input fields users can now clearly read the labels and use the keyboard to navigate the inputs.
```

Comment:
Paradoxically, from the screenshot itself, it now looks with less contrast (the original button was reddish, so more eye catching). Perhaps it'd be better to display these screenshots with the plugin that outlines the problematic elements, and compare the before and after situation?

![image](https://user-images.githubusercontent.com/37327689/189380139-1b2b4878-6322-4161-8791-8d7107788546.png)